lk2nd/device/dts/msm8916/msm8916-wiko-chuppito.dts

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
```

I'd prefer to keep new submissions to BSD-3-clause in lk2nd where possible, this aligns with current upstream linux preference for dts too.